EpikCord/__init__.py

```diff
@@ -36,7 +36,7 @@
 
 from aiohttp import ClientSession, ClientResponse, ClientWebSocketResponse
 
-from .__main__ import __version__
+__version__ = "0.5.2"
```

Comment:
Dont put `__version__` in `__main__` smh, put it in `__init__`. The `__main__` is the package entry point not something you should store importables in.